SU-HSDO/suhumsci
# SHS 5095 Theming User Login Page

# READY FOR REVIEW

## Summary
This PR disables the module the uses the admin theme for the user login page and adjusts the theming to improve the page's appearance.

## Urgency
medium

## Steps to Test
1. Visit the user login page, like https://hs-colorful.suhumsci.loc/user/login
2. Ensure that the page is using the custom theme, and not Gin Admin theme. 
3. Ensure the there is only one title, the box-shadows have been removed, and the title is black (and not a light gray.) 

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
